mail-client/mailspring-bin/mailspring-bin-1.8.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit unpacker
+
+DESCRIPTION="A beautiful, fast and fully open source mail client for Mac, Windows and Linux."
+
+HOMEPAGE="https://getmailspring.com/"
+
+SRC_URI="https://github.com/Foundry376/Mailspring/releases/download/1.8.0/mailspring-1.8.0-amd64.deb"
```

Comment:
Replace 1.8.0 with ${PV} to make version bumping easier.

Reply:
How is` ${PV}` determined ? from the filename ?

Reply:
Yes, see here for an overview of all possible variables: https://devmanual.gentoo.org/ebuild-writing/variables/

Reply:
Good, thank you! I have those refrences in bookmark :D